talentsearch/resources/js/components/search/SearchForm.tsx

```diff
@@ -46,7 +45,7 @@ const FilterBlock: React.FunctionComponent<{
   );
 };
 
-function mapIdToValue<T extends { id: Id }>(objs: T[]): Map<Id, T> {
```

Comment:
Oops, it was being used! 😆 Oh well, I don't really miss it.

Reply:
I can add it back 😆 if you want